src/main/java/org/folio/rest/impl/BaseApi.java

```diff
@@ -49,12 +49,13 @@ public Response buildResponseWithLocation(String okapi, String endpoint, Object
   }
 
   public Void handleErrorResponse(Handler<AsyncResult<Response>> asyncResultHandler, Throwable t) {
+    log.error("Exception encountered", t);
```

Comment:
this is not necessary because we have the same log in Response buildErrorResponse(Throwable throwable)

Reply:
removed, thanks

Reply:
removed, thanks